lib/src/main/java/graphql/nadel/Nadel.kt

```diff
@@ -38,15 +41,12 @@ import java.util.concurrent.atomic.AtomicReference
 import java.util.function.Function
 
 class Nadel private constructor(
-    private val engine: NadelExecutionEngine,
+    private val engine: NextgenEngine,
     val services: List<Service>,
     val engineSchema: GraphQLSchema,
     val querySchema: GraphQLSchema,
-    internal val serviceExecutionFactory: ServiceExecutionFactory,
-    internal val instrumentation: NadelInstrumentation,
-    internal val serviceExecutionHooks: ServiceExecutionHooks,
-    internal val preparsedDocumentProvider: PreparsedDocumentProvider,
-    internal val executionIdProvider: ExecutionIdProvider,
```

Comment:
Removed some things that are no longer needed in this class. Most of the things are now directly passed down to `NextgenEngine`